src/DIPS.Xamarin.UI/Controls/Toast/ToastCore.cs

```diff
@@ -72,8 +72,21 @@ private Grid GetToastContainer()
                     toastContainer.Children.Remove(oldToast);
                 }
             }
-            else // no toast container
+            else // no registered toast container
             {
+                //Get existing toast container 
+                var componentsList = m_currentPageWithToast.LogicalChildren.Where(w => w.GetType() == typeof(ToastContainer)).ToList();
+                if (componentsList.Any())
+                {
+                    var toastContainerWrapper = (ToastContainer)componentsList.First();
```

Comment:
Just do a `FirstOrDefault`, easier to read rather than a `Any` that grabs `First` :)